therapy/schemas.py

```diff
@@ -152,7 +153,6 @@ class ProhibitedSources(Enum):
     """
 
     CHEMBL = SourceName.CHEMBL.value
```

Comment:
Wasn't sure if it was worth keeping this now that it only has one member

Reply:
I think it's fine to keep